docs/site/content/posts/08-build-deploy-apps-w-tce.md

```diff
@@ -0,0 +1,694 @@
+{{< youtube id="KdR_QGXdCzw" title="Build and Deploy an Application with VMware Tanzu Community Edition" >}}
+
+## Introduction
+
+In this demo, we’re going to use Tanzu Community Edition to build and deploy an application. We are using Tanzu Community Edition in a few different ways:
+
+1. To provision a Kubernetes cluster
+2. To enhance that cluster with additional tooling for building, publishing, and storing container images, and for managing applications at runtime
+3. To actually run our application
+
+Let’s get started.
+
+## Cluster provisioning
+
+You can see I actually have several clusters here:
+
+```shell
+$ kubectx
+kind-tce-demo
+tce-mgmt-cluster-admin@tce-mgmt-cluster
+tce-workload-cluster-1-admin@tce-workload-cluster-1
+```
+
+All of these were provisioned with Tanzu Community Edition. This first one (kind-tce-demo) is an example of an unmanaged cluster. This particular one happens to be running on Docker, locally on my machine, similar to how you might run kind or minikube.
+
+The next one is a management cluster (tce-mgmt-cluster), which is useful for use cases where you need to provision many clusters. I’ve used this management cluster, in fact, to provision the third “workload” cluster that you see here (tce-workload-cluster-1). These happen to be running on AWS.
+
+You can see by the color of the output that my context is pointing to the workload cluster. This is the one I will be using for this demo.
+
+## Documentation
+
+It’s probably worth mentioning that all of the documentation is right here on the Tanzu Community Edition website at [tanzucommunityedition.io/docs](https://tanzucommunityedition.io/docs/). Here you’ll find instructions for provisioning both managed and unmanaged clusters.
+
+Tanzu Community Edition also includes a repository of packages you can add, and this documentation includes installation and configuration for each package. This is really great because one of the challenges in building a more comprehensive platform out of a Kubernetes cluster is all of the research and learning and the choices you need to make about additional tooling you might need. Tanzu Community Edition simplifies this process by providing a curated repository of packages and documentation on dependencies, configuration, installation, and some basic usage.
+
+## Cluster namespaces
+
+Let’s take a look at the namespaces in our workload cluster.
+
+```shell
+$ kubens
+cert-manager
+default
+external-dns
+harbor
+kube-node-lease
+kube-public
+kube-system
+projectcontour
+tanzu-package-repo-global
+tkg-system
+tkg-system-public
+```
+
+You can see some namespaces with tkg in the name. This stands for Tanzu Kubernetes Grid and these namespaces contain resources related to core functionality of Tanzu Community Edition, like provisioning clusters, for example.
+
+There is also a namespace called tanzu-package-repo-global—that's where the package repository that I mentioned is installed.
```

Comment:
s/tanzu-package-repo-global—that's/tanzu-package-repo-global - that's/

maybe?